Test/Test.cs

```diff
@@ -33,33 +34,130 @@ public void TestApplyDecisionRequest()
+                                birth_date = "1985-01-19",
+                                loyalty_program = "skymiles",
+                                loyalty_program_id = "PSOV34DF",
+                                phone = "1-415-555-6040",
+                                email = "jdoe@domain.com"
+                            },
+                            new Guest()
+                            {
+                                name = "John Doe"
+                            }
+                        },
+                        segments = new ObservableCollection<Segment>()
+                        {
+                            new Segment()
+                            {
+                                departure_address = new Address
+                                {
+                                    name = "Bill Jones",
+                                    phone =  "1-415-555-6040",
+                                    address_1 = "2100 Main Street",
+                                    address_2 = "Apt 3B",
+                                    city = "New London",
+                                    region = "New Hampshire",
+                                    country = "US",
+                                    zipcode = "03257"
+                                },
+                                arrival_address = new Address
+                                {
+                                    name = "Bill Jones",
+                                    phone =  "1-415-555-6040",
+                                    address_1 = "2100 Main Street",
+                                    address_2 = "Apt 3B",
+                                    city = "New London",
+                                    region = "New Hampshire",
+                                    country = "US",
+                                    zipcode = "03257"
+                                },
+                                start_time = 203841290300L,
+                                end_time = 2038412903,
+                                vessel_number = "LH454",
+                                fare_class = "Premium Economy",
+                                departure_airport_code = "SFO",
+                                arrival_airport_code = "LAS"
+                            }
+                        },
+                        location = new Address
+                        {
+                            name = "Bill Jones",
+                            phone =  "1-415-555-6040",
+                            address_1 = "2100 Main Street",
+                            address_2 = "Apt 3B",
+                            city = "New London",
+                            region = "New Hampshire",
+                            country = "US",
+                            zipcode = "03257"
+                        },
+                        tags = new ObservableCollection<string>() { "tag-123", "tag-321" }
```

Comment:
Why are we using an `ObservableCollection` here? If we don't need the callback, we can probably use something else and avoid having to pull in `ObjectModel`.

Reply:
I have to use ObservableCollection  as NJsonSchema is creating the following section of code in Booking.cs:

```
        [Newtonsoft.Json.JsonProperty("$tags", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
        public System.Collections.ObjectModel.ObservableCollection<string> tags { get; set; }
```

Reply:
ah gotcha. So the only reason it wasn't pulled in before was because our tests were pretty bare-bones and didn't create any other arrays in our test events. Cool, thanks for clarifying.

Reply:
yes, you are right.